pom.xml

```diff
@@ -59,7 +59,7 @@
         <dependency>
             <groupId>uk.gov.ons.ctp.product</groupId>
             <artifactId>casesvc-api</artifactId>
-            <version>10.49.17</version>
+            <version>10.49.23</version>
```

Comment:
Is there a reason this doesn't match the version that Case SVC is using?

Reply:
When my PR goes in on casesvc-api I will update this. It is dependent on that one.